velox/common/encode/Base64.h

```diff
@@ -22,27 +22,37 @@
 #include <folly/Range.h>
 #include <folly/io/IOBuf.h>
 
+#include "velox/common/base/Exceptions.h"
+
 namespace facebook::velox::encoding {
 
-class Base64Exception : public std::exception {
- public:
-  explicit Base64Exception(const char* msg) : msg_(msg) {}
-  const char* what() const noexcept override {
-    return msg_;
+const size_t kCharsetSize = 64;
+const size_t kReverseIndexSize = 256;
+
+/// Character set used for encoding purposes.
+/// Contains specific characters that form the encoding scheme.
+using Charset = std::array<char, kCharsetSize>;
```

Comment:
Is there a specific reason to move this outside of the Base64 class? Keeping it inside the class avoids conflicts with other headers.
Same below for ReverseIndex

Reply:
Let's limit the changes to fixing the code according to the coding guidelines. We can make this change when we create the utility class.

Reply:
Redid the code